src/integrationtest/java/com.microsoft.aad.msal4j/ApacheHttpClientAdapter.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import org.apache.http.Header;
+import org.apache.http.client.methods.CloseableHttpResponse;
```

Comment:
nit: license

Reply:
License header? think you might have missed it, it's already at the top of this file